pqdb-frontend/README.md

```diff
@@ -0,0 +1,33 @@
+# PQDB Frontend
+
+This is the frontend for displaying data from pqdb. The latest deployment of the frontend can be accessed at https://cryptoeng.github.io/pqdb/.
+
+## Testing locally
+
+
+### Requirements
+
+* Working schema validation script (For install instructions see [here](/README.md) under Schema Validation)
+* Yarn (Installs with node.js, if you use the snap package. Otherwise see [here](https://classic.yarnpkg.com/en/docs/install/))
+
+Before you can run the frontend locally you need to generate the SQLite database and the database diagram. This can be done by running the following command from the root directory of pqdb.
+```
+node tools/validation/validate.js pqdb-frontend/public/pqdb.sqlite pqdb-frontend/src/tables.svg
+```
+If you want to update the data source for the frontend, you need to remove the two files and re-run the command.
+
+### Installation
+If not done already, switch to the `tools/pqdb-frontend` directory, then install the dependencies.
+
+```
+yarn install
+```
+
+
+### Run
+
+```
+yarn start
+```
+
+This starts the React app and loads the url in the browser.
```

Comment:
A little SQL example would be convenient. Especially because the table overview is a little overwhelming at first

Reply:
This is the framework for the frontend views, so SQL examples will come soon, when views are created.